src/components/ListOptions/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FunctionComponent} from 'react'
```

Comment:
I know we are following the same concept throughout the codebase, but we should really be loading react as
`import * as React from 'react`

Reply:
Why do you want to import things that aren't used in the code?  Most IDEs will complain about unused variables.  Plus it's opaque as to what names are now in your namespace.

IMO